lua/entities/gmod_wire_expression2/base/tokenizer.lua

```diff
@@ -145,44 +145,35 @@ function Tokenizer:NextSymbol()
 
 	elseif self:NextPattern("^[a-z#][a-zA-Z0-9_]*") then
 		-- keywords/functions
-		if self.tokendata == "if" then
-			tokenname = "if"
-		elseif self.tokendata == "elseif" then
-			tokenname = "eif"
-		elseif self.tokendata == "else" then
-			tokenname = "els"
-		elseif self.tokendata == "local" then
-			tokenname = "loc"
-		elseif self.tokendata == "while" then
-			tokenname = "whl"
-		elseif self.tokendata == "for" then
-			tokenname = "for"
-		elseif self.tokendata == "break" then
-			tokenname = "brk"
-		elseif self.tokendata == "continue" then
-			tokenname = "cnt"
-		elseif self.tokendata == "switch" then
-			tokenname = "swh"
-		elseif self.tokendata == "case" then
-			tokenname = "case"
-		elseif self.tokendata == "default" then
-			tokenname = "default"
-		elseif self.tokendata == "foreach" then
-			tokenname = "fea"
-		elseif self.tokendata == "function" then
-			tokenname = "func"
-		elseif self.tokendata == "return" then
-			tokenname = "ret"
-		elseif self.tokendata == "void" then
-			tokenname = "void"
-		elseif self.tokendata == "#include" then
-			tokenname = "inclu"
+		local keywords_and_tokens = {
+			["if"] = "if",
+			["elseif"] = "eif",
+			["else"] = "els",
+			["local"] = "loc",
+			["while"] = "whl",
+			["for"] = "for",
+			["break"] = "brk",
+			["continue"] = "cnt",
+			["switch"] = "swh",
+			["case"] = "case",
+			["default"] = "default",
+			["foreach"] = "fea",
+			["function"] = "func",
+			["return"] = "ret",
+			["void"] = "void",
+			["#include"] = "inclu"
+		}
+
+		if keywords_and_tokens[self.tokendata] ~= nil then
+			tokenname = keywords_and_tokens[self.tokendata]
 		elseif self.tokendata:match("^[ijk]$") and self.character ~= "(" then
-			tokenname, self.tokendata = "num", "1" .. self.tokendata
+			tokenname = "num"
```

Comment:
This change has nothing to do with PR's intention. Why'd you split a single assignment operation into two...

Reply:
I think having multiple assignments on one line is pointless just to save one LOC, plus it was more of a force of habit (Python PEP8 style). The intention of this PR is to improve readability and maintainability